Pima indians diabetes and linear classifier/Заключение/task.md

```diff
@@ -0,0 +1,7 @@
+In this lesson, we've discussed an example of implementing the linear classifier utilizing the stochastic gradient descent.
+
+Among all local optimization methods, this method is the easiest in implementation. Nevertheless, it is a powerful tool for tasks of this kind. We've considered a classification example with two classes – the presence or absence of type 2 diabetes. However, the algorithm may be used with more classes, too, even if then it will require considerable modifications.
```

Comment:
maybe: easiest -> simplest?

Reply:
easiest is better here (like - without effort)
The sentence – yes, yours is good.

Reply:
ok!